LabApiUtilities/build.gradle

```diff
@@ -76,3 +80,39 @@ afterEvaluate {
     // above in the plugins block.
     assemble.dependsOn generateBuildConfig, generateTestBuildConfig
 }
+
+jar {
+    manifest {
+        attributes('Implementation-Title': project.name,
+                'Implementation-Version': project.version)
+    }
+}
+
+publishing {
+    publications {
+        aar(MavenPublication) {
+            groupId 'com.microsoft.identity'
+            artifactId 'LabApiUtilities'
```

Comment:
let's call it `lab-api-utilities` (I believe this is a more standard pattern for naming artifact ids)

Reply:
88a34f4e372f037866173370dff119603a78f482 solved